internal/pkg/cli/svc_init.go

```diff
@@ -393,7 +393,7 @@ func (o *initSvcOpts) askImage() error {
 		return nil
 	}
 
-	var validator prompt.ValidatorFunc
+	validator := validateNonEmptyString
```

Comment:
sorry my bad - I should have let you know! I think we have a validator already in the `prompt` pkg: 
https://github.com/aws/copilot-cli/blob/cc94d5ea457366a738dcb347468b29b09876b46f/internal/pkg/term/prompt/prompt.go#L343-L345